ckanext/opendata_theme/opengov_custom_header/templates/admin/custom_header.html

```diff
@@ -0,0 +1,119 @@
+{# The site to config site styles, accessibile to sysadmins#}
+
+{% extends "admin/base.html" %}
+
+{% import 'macros/form.html' as form %}
+
+{% set custom_header_action = h.url_for(
+  controller='ckanext.opendata_theme.opengov_custom_header.controller:CustomHeaderController',
+  action='custom_header') %}
+
+{% set reset_custom_header = h.url_for(
+  controller='ckanext.opendata_theme.opengov_custom_header.controller:CustomHeaderController',
+  action='reset_custom_header') %}
+
+{% set remove_link_from_custom_header = h.url_for(
+  controller='ckanext.opendata_theme.opengov_custom_header.controller:CustomHeaderController',
+  action='remove_link') %}
+
+{% set add_link_from_custom_header = h.url_for(
+  controller='ckanext.opendata_theme.opengov_custom_header.controller:CustomHeaderController',
+  action='add_link') %}
+
+{% block styles %}
+    {{ super() }}
+    {% resource 'opengov_custom_theme_resource/main' %}
+    {% resource 'opengov_custom_header_resource/main' %}
+{% endblock %}
+
+{% block primary_content_inner %}
+    {{ form.errors(errors) }}
+    <form method="post" action="{{ add_link_from_custom_header }}">
+        <h3>Add Navigation Link</h3>
+            <div class="form-inline row">
+                <div class="form-group col-sm-4 span4">
+                    <div class="input-group">
+                        <label for="new_title"> Title </label>
+                        <input type="text" class="form-control" id="new_title" name="new_title"/>
+                    </div>
+                </div>
+                <div class="form-group col-sm-5 span5">
+                    <div class="input-group">
+                        <label for="new_link"> Link </label>
+                        <input type="text" class="form-control" id="new_link" name="new_link"/>
+                    </div>
+                </div>
+            </div>
+        <button class="btn-default" name="add" type="submit">{{ _('Add') }}</button>
+    </form>
+    <form method="post" action="{{ remove_link_from_custom_header }}">
+        <h3>Remove Navigation Link</h3>
+        <div class="form-group form-inline">
+            <select class="col-md-6 span6" name="to_remove">
+                {% for link in links %}
+                    <option value="{{ link.title }}">{{ link.title }} ({{ link.link }})</option>
+                {% endfor %}
+            </select>
+        </div>
+        <div class="form-group form-inline">
+            <button class="btn-default" name="remove" type="submit">{{ _('Remove') }}</button>
+        </div>
+    </form>
+    <form method="post" action="{{ custom_header_action }}" id="custom-header-form">
+        <div class="control-group">
+            <div class="controls">
+                <h3 >Header Layout</h3>
+                <label class="checkbox inline">
+                    <input class="form-check-input" type="radio" name="layout_type" value="default"  {% if layout_type=='default' %}checked {% endif %}>Default
```

Comment:
It may be a good idea to set layout to default if layout_type is not set. This would be the case if the extension is newly enabled.

Reply:
updated